### PR TITLE
Fix default value display

### PR DESCRIPTION
**Pull request by SpacePossum**
before
after

### CODE REVIEW

**Comment by SpacePossum** [comment]
>what was wrong ?

the partly yellow `|` in the display

File: README.rst [hunk 1]
```diff
@@ -221,7 +221,10 @@ Choose from the list of available rules:
   Configuration options:
 
   - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array
-    syntax; defaults to ``'long'``
+    syntax; defaults to:
+    ``
```
[the remark this answers is not in the document]
Comment (SpacePossum) [reply README.rst]: thanks, should be fixed now :)

**Comment by SpacePossum** [comment]
before_2
after_2

3rd commit is word wrap for long risky descriptions

File: README.rst [hunk 2]
```diff
@@ -273,13 +278,16 @@ Choose from the list of available rules:
   Configuration options:
 
   - ``multi_line_extends_each_single_line`` (``bool``): whether definitions should
-    be multiline; defaults to ``false``; DEPRECATED alias:
+    be multiline; defaults to:<default_block>; DEPRECATED alias:
```
Comment (SpacePossum) [inline README.rst]: note to self: this tag should not end up in the `README.rst`

Comment (SpacePossum) [reply README.rst]: fixed

**Comment by SpacePossum** [comment]
Sadly we have the same `yellow |` issue for subset listings as well (didn't spotted it the first time around)

```
     | Configuration options:
     | - functions (a subset of ['array_key_exists', 'empty', 'file_exists',
```

I let it out of scope for now

**Comment by SpacePossum** [comment]
So now the output in the console looks a lot better IMHO, but the README markdown is a bit less good than it was.... we should really decouple those ...

**Comment by keradus** [comment]
not sure about the flags here.
`easy pick` - you want someone else to take over this PR?
`help wanted` - sth in particular, eg some failing test?

**Comment by SpacePossum** [comment]
>help wanted

with patching up this PR so the README looks good as well

>easy pick

cause it shouldn't be hard (I just don't have the time to fix it myself ATM)

**Comment by julienfalque** [comment]
> So now the output in the console looks a lot better IMHO, but the README markdown is a bit less good than it was.... we should really decouple those ...

I'm planning to reduce README file drastically in #3475 and its content will not be tied to help of `fix` command anymore. Maybe you want me to put my PR in the top of my priority list?

**Comment by SpacePossum** [comment]
3475 looks very promising to me, I wished I've spend more time on reviewing it!
After the next round of releases, lets join up, you and me and others who want to (I can image some people working on doc generation as well), and get this thing done! been on my wish list for a long time so I'm in if you are ;)

**Comment by SpacePossum** [comment]
TODO: restore line break before and after `   | Configuration options:\n` in the `ReadmeCommand` and this thing should be done

**Comment by SpacePossum** [comment]
I think this looks good now

**Comment by julienfalque** [comment]
Support for branch 2.2 has ended, please rebase on 2.12.

File: src/Console/Command/HelpCommand.php [hunk 3]
```diff
@@ -400,7 +400,11 @@ public static function getLatestReleaseVersionFromChangeLog()
      */
     protected function initialize(InputInterface $input, OutputInterface $output)
     {
-        $output->getFormatter()->setStyle('url', new OutputFormatterStyle('blue'));
+        $formatter = $output->getFormatter();
+        $formatter->setStyle('url', new OutputFormatterStyle('blue'));
+        $formatter->setStyle('risky', new OutputFormatterStyle('black', 'yellow'));
+        $formatter->setStyle('default_block', new OutputFormatterStyle('white'));
+        $formatter->setStyle('default_line', new OutputFormatterStyle('yellow'));
```
Comment (keradus) [inline src/Console/Command/HelpCommand.php]: shouldn't we add this for `DescribeCommand` as well ?

**Comment by SpacePossum** [comment]
Amount of needed changes, which not all look so good, are to much for just the `yellow |` issue.
I might puzzle something smaller together in the near future.

**Pull request closed without merging**